webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/reports/ExecutionBatchTest.scala

```diff
@@ -1444,14 +1444,15 @@ class ExecutionBatchTest extends Specification {
 
   }
 
+  // see correction with report id in next test
   "Sub block with looping component and same component names are not correctly reported, see https://issues.rudder.io/issues/20071" should {
     val reportsWithLoop = Seq[ResultReports](
       new ResultRepairedReport(
         executionTimestamp,
         "cr",
         "policy",
         "nodeId",
-        "report_id12",
+        "report_id_b1c1",
```

Comment:
use different reportId just to prove that when we have expected reports not supporting them, they are not used. In the real life, it will be "0" everywhere in that case.